### ISSUE TITLE
Rule L026 bug with simple DELETE statements

### ISSUE THREAD

**Issue by tunetheweb** [sqlfluff/sqlfluff#2457]
<!--If this is a parsing or linting issue, please include a minimal SQL example which reproduces the issue, along with the `sqlfluff parse` output, `sqlfluff lint` output and `sqlfluff fix` output when relevant.-->

## Expected Behaviour

This should give no errors:

```
echo "delete from person where 1 = 1;" | sqlfluff lint -
```


## Observed Behaviour
```
% echo "delete from person where 1 = 1;" | sqlfluff lint -
CRITICAL   [L026] Applying rule L026 threw an Exception: 'NoneType' object has no attribute 'selectables'                                                                                                                                       
Traceback (most recent call last):
  File "/Users/barry/sqlfluff/src/sqlfluff/core/rules/base.py", line 519, in crawl
    res = self._eval(
  File "/Users/barry/sqlfluff/src/sqlfluff/rules/L026.py", line 90, in _eval
    self._analyze_table_references(
  File "/Users/barry/sqlfluff/src/sqlfluff/rules/L026.py", line 116, in _analyze_table_references
    for selectable in query.selectables:
AttributeError: 'NoneType' object has no attribute 'selectables'
== [stdin] FAIL                                                                                                                                                                                                                                 
L:   1 | P:   1 | L026 | Unexpected exception: 'NoneType' object has no attribute
                       | 'selectables'; Could you open an issue at
                       | https://github.com/sqlfluff/sqlfluff/issues ? You can
                       | ignore this exception for now, by adding '-- noqa: L026'
                       | at the end of line 1
```


## Steps to Reproduce

As per above

## Dialect

Any

## Version
<!-- Include the output of `sqlfluff --version` along with your Python version.
If you are using dbt then please additionally include the dbt version. -->
0.9.2

## Configuration

Default
